modules/rosa-sts-associating-your-aws-account.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-associating-your-aws-account_{context}"]
+= Associating your AWS account with your Red Hat organization
+
+Before using {cluster-manager-first} to create {product-title} (ROSA) clusters that use the AWS Security Token Service (STS), create an {cluster-manager} IAM role and link it to your Red Hat organization. Then, create a user IAM role and link it to your Red Hat user account in the same Red Hat organization.
```

Comment:
Should we say (ocm role in brackets for OpenShift Cluster Manager IAM role

Reply:
@pvasant thank you for your review and for this suggestion. I have noticed that the terminology to refer to this role varies throughout the ROSA docs library. In the ROSA installation docs that added in https://github.com/openshift/openshift-docs/pull/45306 previously, "OpenShift Cluster Manager role" and "OpenShift Cluster Manager IAM role" were used. In the About IAM resources for ROSA clusters that use STS section introduce through a separate PR, in some cases the role is referred to as "`ocm-role`".

I would like to separate a review of the naming convention for that role out to a separate Jira, given that this is wider than this PR. To facilitate this, I have created https://issues.redhat.com/browse/OSDOCS-4058. I will keep the terminology as is in this current PR and we will discuss the naming separately. After we have reviewed the terminology, we can update the documentation for consistency.

Please also see [this previous docs PR comment](https://github.com/openshift/openshift-docs/pull/45306#discussion_r882465887) for some discussion on this topic.